Sources/Xcodeproj/pbxproj().swift

```diff
@@ -196,6 +203,16 @@ public func pbxproj(srcroot: AbsolutePath, projectRoot: AbsolutePath, xcodeprojP
         print("            sourceTree = BUILT_PRODUCTS_DIR;")
         print("        };")
 
+        // shell script build phase.
+        if module.isClangModuleLibrary {
+            print("        \(module.shellScriptPhaseReference) = {")
+            print("            isa = PBXShellScriptBuildPhase;")
+            print("            shellPath = /bin/sh;")
+            print("            shellScript = \"mkdir -p $BUILT_PRODUCTS_DIR/SymlinkLibs\nln -sf ../$EXECUTABLE_PATH $BUILT_PRODUCTS_DIR/SymlinkLibs/lib$EXECUTABLE_NAME.dylib\";")
```

Comment:
This needs some quoting so that things don't fall over if they have whitespace in their paths.